gtfs/gtfs_utils/gtfs_utils/core_computations.py

```diff
@@ -23,6 +24,50 @@ def _read_almost_valid_csv_to_df(zip_path, file_name_in_zip, real_columns):
     return df
 
 
+def _fix_trip_to_date_columns(trip_date_df: pd.DataFrame) -> None:
+    """
+    Parse date and time columns
+    and fix rows where departure time is after midnight (hours after 24:00)
```

Comment:
**Do you mean the hours after midnight?** 
   - for departure hours between 00:00 and 03:59 they are using 24:00 to 27:59
**or all the parsing logic?**
    - for each file we need to parse the date and time columns

Reply:
I meant the hours after midnight. 
So it means that always 4<=start_time<28 ?

Reply:
Yes from what I've seen thought I couldn't find any thing about it.
I saw this comment:
![image](https://user-images.githubusercontent.com/36769369/74826383-2c8b4f00-5314-11ea-94c0-58de3a78bc16.png)

but it's about the `arrival_time` column in `stop_times.txt`

Reply:
I know this about the `stop_times.txt`, but I just wonder whether the fix you applied might miss something in rare cases where it is not `4<=start_time<28`. Maybe we should add a sanity check here also? Or at least test on some more files (no need for a full `gtfs_stats` run, just parsing `TripIdToDate` file and checking start_time range)

Reply:
I could try testing on more files, but I'm fixing the time with modulo, so start_time will always between 0 and 24 (even if they reach 29)

Reply:
Yeah I'm mostly worried about date miss-match. You say date is always correct, what happens if it is the last day of ToDate for example and start_time>=24:00? 
for example this line from today's `TripIdToDate.zip`:
```11688,83001,1,7,01/03/2020 00:00:00,01/03/2020 00:00:00,43849192,6,25:30,
```
what date does this refer to?

Reply:
It refers to the next date after `ToDate`.
I had a case like that in one of the days I was testing. the times and day of the week only matched if I matched it to the next day.

In your example it would be 01:30 of the 02/03/2020

Reply:
Also I just tested a whole month (21/01/2019 - 21/02/2019) and the assumption that `4<=start_time<28` still holds. Do you want me to leave the assertion in the code?

Reply:
We can probably drop it, but I like it there for safety :)